### DISCUSSION TITLE
AddressBook

### DISCUSSION THREAD

**Discussion by crawe1** [YetiForceCompany/YetiForceCRM#14617] [How to]
Hi,

It is possible to add annother default group to search in roundcube ? default is only Contacts? 

when I enter the address in the field, it only searches in contacts. 

And last questions it is possible to automatically entered  add new contacts to addressbook? 

![image](https://user-images.githubusercontent.com/61950059/116585333-92139c00-a918-11eb-8cfd-f53c7b409bf3.png)

**Comment by crawe1** [comment]
Ok its simple 
1) Add new workflow task And Check chceckbox with generate addressbook 
2) add in file modules/vtiger/cron/addressbook module name from step 1. 
 
But how add automaticly new e-mail addresses by roundcube after send Message. Someone have any idea ?

**Reply by rskrzypczak** [reply]
> But how add automaticly new e-mail addresses by roundcube after send Message. Someone have any idea ?

Currently there is no such option.

**Reply by crawe1** [reply]
in my opinion a better solution would be to create a module than just one table in the database. cron change to add to the module than directly to table. user could control this by adding an automatic flag for cron operations and a manual flag for user operations. 
Radek can you tell me what is better Create new module with new name and new table and link to round cube ? or change name actual table name and create new module with this same name ?
